src/ConnectAuth.ts

```diff
@@ -22,6 +30,7 @@ export type VerifyInstallationArgs<E> = CommonVerifyArgs<E, InstallationQueryStr
 export interface NewInstallationResponse {
   type: InstallationType.newInstallation;
   clientKey: string;
+  connectJwt?: ConnectJwt;
```

Comment:
With signed installs, installation will have a JWT that can be used to obtain extra information if the consumer desires to do so.